src/components/Collapsible.js

```diff
@@ -0,0 +1,346 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { IoIosArrowDown, IoIosArrowForward } from 'react-icons/io';
+
+/**
+ * This Component was adapted from https://www.npmjs.com/package/react-collapsible.
+ */
+class Collapsible extends Component {
```

Comment:
Why was this taken and not just installed via npm?
I don't really want to maintain this Component if not 100% required.

Reply:
This is a very simple and light Component. On the one side it does not contain dependencies to any other library, besides react ofc. On the other side there's not much of a profit leaving it remotely. This way we have full control and can adjust it, as I just did. Easy example: I want to have arrows indicating whether something is/can be expanded. If I simply install the component I have to implement an ugly workaround, most probably building a component which contains this one.

Reply:
I don't really think wrapping the component is an ugly workaround. It is a pretty common pattern in react.

Applying patches from the upstream component / package would be pretty tedious this way, especially when we have changed the code of it. 
This would be pretty annoying if the component breaks because of changes to React, some Browser or the build process.

Reply:
This might also be problematic with the license of the component as the license requires that the license is included if the component if copied.